.github/workflows/typos.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     - name: Checkout Actions Repository
       uses: actions/checkout@v3
 
-    - name: Check spelling of truelearn
+    - name: Check spelling of Truelearn
```

Comment:
TrueLearn?

Reply:
Yea thats correct, I merged it and then I saw this. 
Ill just force push this change on main